src/lib/css/theme/bridge.scss

```diff
@@ -10,6 +10,6 @@
   --active: rgb(0 0 0 / 0.1);
 }
 
-body.dark {
+body.body--dark {
   --active: rgb(255 255 255 / 0.2);
```

Comment:
For the fix, I was thinking more about using the MD3 colors rather than keeping this `active` variable. What do you think?